contrib/initramfs/scripts/zfs

```diff
@@ -426,6 +429,7 @@ decrypt_fs()
 
 			# Prompt with systemd, if active 
 			elif [ -e /run/systemd/system ]; then
+                                echo -n "systemd-ask-password" > /run/zfs_console_askpwd_cmd
```

Comment:
It appears this is using spaces instead of tabs like the rest of the file.

Reply:
done, replaced spaces with tabs